playbook/roles/prereq/tasks/main.yaml

```diff
@@ -22,7 +22,7 @@
         var: ufw_status
 
     - name: If ufw enabled, open api port
-      when: ufw_status['stdout'] == "Stato':'' attivo"
+      when: ufw_status.stdout | regex_search('attivo')
```

Comment:
This looks language dependent. What about if the operating system is in English?

Reply:
This is an important observation and we should take into account the possibility of having different languages.

The ideal way to do it is to provide scripts that are language-agnostic but this would require some complexity which is not easy to manage (dealing with locale setup).

An alternative, would be to assume systems are always in English and so changing the condition with English labels.

A good compromise is to assume system could be either in English or in Italian (as provided by us).

In this case, we can use conditions like when: `ansible_env.LANG.startswith('it')` or when: `ansible_env.LANG.startswith('en')` to satisfy both cases.

Reply:
I agree with Francesco.
We can make a pre-check in the Ansible script to check the language, and return an error in case the language is different from what we think. Given that we're using mainly Linux, I would choose English as the main language.

Reply:
We had a call this morning and we think we can move this to the next PR as this one mainly focuses on updating the README.
Waiting for any other feedback before squashing and merging.